### ISSUE TITLE
DCAT-US non-federal harvests rejected on federal validation

### ISSUE THREAD

**Issue by jbrown-xentity** [GSA/data.gov#2988]
Example: [City of Sioux Falls Data.json](https://admin-catalog-next.data.gov/harvest/admin/097b647c-9eb8-426b-b39a-e8a57b496af5) (there are more)

The email reports the harvest config as `{"default_groups": "local", "validator_schema": "non-federal", "private_datasets": "False"}`

The [harvest validation code](https://github.com/GSA/ckanext-datajson/blob/master/ckanext/datajson/datajson_ckan_28.py#L304-L331) seems setup appropriately. Something is wrong/missing.

## How to reproduce

1. Reharvest [City of Sioux Falls Data.json](https://admin-catalog-next.data.gov/harvest/admin/097b647c-9eb8-426b-b39a-e8a57b496af5)
1. Review harvest errors once complete

## Expected behavior

Input data validated against given schema (in example case, `non-federal`

## Actual behavior

Given schema ignored; data validated against `federal` schema.

## Sketch
Test harvest locally with debug statements

**Comment by mogul** [comment]
Likely dying because non-federal sources are failing a schema check. There are probably a lot of others in the same situation.

**Comment by jbrown-xentity** [comment]
Noticed a specific example, WA JSON harvest source email. Already had catalog up locally, so tested locally on same harvest source.
All 1K objects were harvested successfully.
Not sure if this is because my catalog build is an old version, or if this works locally and something is wrong with the harvest source config on production...
Example email we've gotten:

```
This is a failure notification of the latest harvest job set-up in https://catalog.data.gov.
Job URL: https://catalog.data.gov/harvest/1c33d1cb-2f69-4f1b-835e-453790f38dc7/job/3f2cd588-15e8-4338-b4b4-ba397ab3d512

Harvest Source: WA JSON
Harvest Configuration: {"default_groups": "local", "validator_schema": "non-federal", "private_datasets": "False"}
Organization: state-of-washington

Harvest Job Id: 3f2cd588-15e8-4338-b4b4-ba397ab3d512
Created: 2021-03-19 04:27:39.440471
Finished: 2021-03-19 19:04:52.225096

Records in Error: 1035
Records Added: 0
Records Updated: 0
Records Deleted: 0

Error Summary: 20 errors

Job Errors: 0

Document Error: 20

    - Identifier: https://data.wa.gov/api/views/aayx-fa6z; Title: Report Card Teacher Qualification Grade Level 2019-20 School Year; 2 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property.
    - Identifier: https://data.wa.gov/api/views/jcbe-f4aj; Title: Bainbridge High School; 2 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property.
    - Identifier: https://data.wa.gov/api/views/e3x6-7wqn; Title: Adult Abundance Recovery Goals (2020 SOS); 3 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;description&#39;:&#39;&#39; is too short.
    - Identifier: https://data.wa.gov/api/views/9nhq-knvs; Title: Shoreline Childrens Center; 2 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property.
    - Identifier: https://data.wa.gov/api/views/rk5w-3q8b; Title: Report Card Student Access To Educators Summary 2017-18 School Year; 2 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property.
    - Identifier: https://data.wa.gov/api/views/vsr8-3iup; Title: 1 -- All $$ By Biennium; 4 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;keyword&#39; is a required property;  ### ERROR #4: &#39;description&#39;:&#39;&#39; is too short.
    - Identifier: https://data.wa.gov/api/views/uwaz-vdsi; Title: Hanford High School; 2 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property.
    - Identifier: https://data.wa.gov/api/views/kjiu-4cue; Title: West Hill STEMS; 2 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property.
    - Identifier: https://data.wa.gov/api/views/k8rh-64ja; Title: Neah Bay High School; 2 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property.
    - Identifier: https://data.wa.gov/api/views/v7ua-cvix; Title: MSP-HSPE Scores by School_2014; 3 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;description&#39;:&#39;&#39; is too short.
    - Identifier: https://data.wa.gov/api/views/d5dm-bwe8; Title: Report Card Graduation 2014-15; 2 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property.
    - Identifier: https://data.wa.gov/api/views/59z5-2b83; Title: EOC Scores by State_2012; 3 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;description&#39;:&#39;&#39; is too short.
    - Identifier: https://data.wa.gov/api/views/2uyp-yrqf; Title: K-12 Public School Enrollment by Grade Level October Enrollment Report 2009-2013; 4 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;keyword&#39; is a required property;  ### ERROR #4: &#39;description&#39;:&#39;&#39; is too short.
    - Identifier: https://data.wa.gov/api/views/g383-z3j8; Title: WLPT-II Scores by School_2011; 3 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;description&#39;:&#39;&#39; is too short.
    - Identifier: https://data.wa.gov/api/views/7na8-mwsc; Title: Report Card Growth for 2014-15; 3 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;description&#39;:&#39;&#39; is too short.
    - Identifier: https://data.wa.gov/api/views/iha6-a4x9; Title: AYP Data by District - participation-other indicator_2010; 3 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;description&#39;:&#39;&#39; is too short.
    - Identifier: https://data.wa.gov/api/views/yamm-p8xi; Title: WaKids Scores by State_2015; 3 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;description&#39;:&#39;&#39; is too short.
    - Identifier: https://data.wa.gov/api/views/82ck-bmn3; Title: State WASL Scores by Subgroup_2005; 3 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;description&#39;:&#39;&#39; is too short.
    - Identifier: https://data.wa.gov/api/views/c4ag-3cmj; Title: Lobbyist Summary; 2 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property.
    - Identifier: https://data.wa.gov/api/views/necc-m8mq; Title: Choose Washington Site Selector; 3 Error(s) Found. ### ERROR #1: &#39;bureauCode&#39; is a required property;  ### ERROR #2: &#39;programCode&#39; is a required property;  ### ERROR #3: &#39;distribution&#39;:[{&#39;downloadURL&#39;: &#39;http://choosewashingtonstate.com/i-need-help-with/site-selection/&#39;, &#39;title&#39;: &#39;Choose Washington Site Selector launch page&#39;, &#39;@type&#39;: &#39;dcat:Distribution&#39;, &#39;mediaType&#39;: &#39;text/html&#39;}, {&#39;downloadURL&#39;: &#39;http://washington.zoomprospector.com/main/tools/SearchFeeds.ashx?SST=WASHINGTON&amp;Q={%22SearchType%22:%22BUILDINGS%22,%22PropertyID%22:null,%22BrokerID%22:null,%22ExternalID%22:null,%22PropertyType%22:%22%22,%22MinSize%22:null,%22MaxSize%22:null,%22GeoEntityList%22:%22%22,%22RegionsList%22:%22%22,%22Attributes%22:%22%22,%22Address%22:null,%22IsBuilding%22:true,%22PolyPoints%22:%22%22,%22SessionID%22:%2283c0d68d-0e07-3685-8315-8a3c7ce47e7d%22,%22SubsetToken%22:%22WASHINGTON%22,%22StartRowID%22:27,%22EndRowID%22:52,%22SortBy%22:%22default%22,%22SortDirection%22:false,%22RequestID%22:%229f403d57-6ad0-624a-0cb4-b2f6f1c530b8%22,%22InputParameters%22:null,%22Keywords%22:%22%22,%22ZipCode%22:%22%22,%22BrokerList%22:%22%22,%22Company%22:null,%22ManagerID%22:null,%22Removed%22:null}&#39;, &#39;mediaType&#39;: &#39;application/rss+xml&#39;, &#39;title&#39;: &#39;Available Commercial Properties&#39;, &#39;@type&#39;: &#39;dcat:Distribution&#39;, &#39;description&#39;: &#39;This is a user generated feed based on search criteria&#39;}] is not valid under any of the given schemas.
```

**Comment by FuhuXia** [comment]
Duplicate of https://github.com/GSA/datagov-deploy/issues/3297.
